vagrant.rb

```diff
@@ -90,15 +120,15 @@ def configure(config, os, pf_name, pf_id, host_name, host_id,
 
   # provisioning script
   if os == $windows7 or os == $windows2008 then
-    command = "c:/vagrant/scripts/network.cmd #{net} #{host_list}\n"
+    command = "c:/vagrant/scripts/network.cmd #{net} @host_list@\n"
```

Comment:
nothing, it is a marker
The goal is to avoid interpreting this variable now because it is not yes full.
It is interpreted at the bottom of the script with a sub function